src/main/java/com/ita/if103java/ims/util/ListUtils.java

```diff
@@ -28,4 +28,8 @@
                 .flatMap(x2 -> predicate.test(x1, x2) ? Stream.of(zipper.apply(x1, x2)) : Stream.empty()))
             .collect(Collectors.toList());
     }
+
+    public static boolean isNullOrEmpty(List<?> list) {
+        return list == null || list.isEmpty();
```

Comment:
You can import and use the following
CollectionUtils.isEmpty()
https://commons.apache.org/proper/commons-collections/apidocs/org/apache/commons/collections4/CollectionUtils.html